python/ray/_private/ray_option_utils.py

```diff
@@ -188,7 +188,7 @@ def issubclass_safe(obj: Any, cls_: type) -> bool:
         if (x is None or x == "dynamic" or x == "streaming" or x >= 0)
         else "The keyword 'num_returns' only accepts None, a non-negative integer, or "
         '"dynamic" (for generators)',
-        default_value=1,
+        default_value=None,
```

Comment:
If the function is a generator, num_returns="streaming", otherwise it will become 1 in the remote function layer